File: Java/Types of Matrix/Transpose matrix.java
class transpose_matrix {
    public static void main(String[] args) {
        // Transpose matrix is interchanging its rows into columns or columns into rows (A = A' (or A^T))
        int[][] A = {
            { 1, 2, 3 },
            { 4, 5, 6 },
            { 7, 8, 9 }
        };
        int[][] B = new int[A[0].length][A.length];

        System.out.println("Matrix: ");
        // Transpose here to B[][] using for loop
        for (int row = 0; row < A.length; row++) {
            for (int col = 0; col < A[row].length; col++) {
                B[col][row] = A[row][col];
                System.out.print("\t" + A[row][col]);
            }
            System.out.println();
        }

        // Print transposed matrix
        System.out.println("\nTranspose matrix: ");
        for (int row = 0; row < B.length; row++) {
            for (int col = 0; col < B[row].length; col++) {
                System.out.print("\t" + B[row][col]);
            }
            System.out.println();
        }
    }
}

File: Python/Types of Matrix/Transpose matrix.py
# Transpose matrix is interchanging its rows into columns or columns into rows (A = A' (or A^T))
A = [[1, 2, 3],
     [4, 5, 6]]

B = [[None]*len(A) for row in range(len(A[0]))]

print("Matrix: ")
# Transpose here to B[][] using for loop
for row in range(0, len(A)):
    for col in range(0, len(A[row])):
        B[col][row] = A[row][col];
        print("\t", A[row][col], end='')
    print()

# Print transposed matrix
print("\nTranspose matrix: ")
for row in range(0, len(B)):
    for col in range(0, len(B[row])):
        print("\t", B[row][col], end='')
    print()